index.html

```diff
@@ -0,0 +1,332 @@
+            </div>
+
+            <div class="tags">
+                <span class="tag">html</span>
+                <span class="tag">css</span>
+                <span class="tag">javascript</span>
+            </div>
+            <div class="action">
+                <button type="button" class="btn"><span>See Project</span> </button>
+            </div>
+        </div>
+        </div>
+        <!---------End of caard-works for multi-post stories-2------------->
+
+        <!----end of container div------------->
+        </section>
+        <!-----section-work ends here------->
+
+        <!---------About myself section------->
+        <section id="about" class="section-about-myself">
+            <div class="card-works about-myself">
+                <div class="welcome">
+                    <h1 class="name-welcome">
+                        About <br> Myself
+                    </h1>                
+                </div>
+                <div class="job-description">
+                    <p class="description-of-me">
+                        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                    </p>
+                </div>
+                
+                <div class="lets-connect lets-connect-about">
+                    <p class="lets-connect-statement">Let's connect</p>
+    
+    
+                    <ul class="social-networks about-myself-social-ntks">
+                        <li>
+                            <a href="#" class="social-network">
+                                <i class="fa fa-twitter"></i>
+                            </a>
+                        </li>
+                        
+                        <li>
+                            <a href="#" class="social-network">
+                                <i class="fa fa-linkedin"></i>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="social-network">
+                                <i class="fa fa-medium"></i>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="social-network">
+                                <i class="fa fa-github"></i>
+                            </a>
+                        </li>
+                       <li>
+                        <a href="#" class="social-network">
+                            <i class="fa fa-angellist"></i>
+                        </a>
+                       </li>
+                    </ul>                
+                </div>
+                <div class="action">
+                    <button type="button" class="btn about-me-btn"><span>Get my resume</span> </button>
+                </div>
```

Comment:
- [ ] Kindly style this button to match the Figma design, please

*Figma design*
![btn](https://user-images.githubusercontent.com/94551484/186519824-4f6d22f8-4369-415e-90f5-dc698b2c979d.png)
.